x-pack/plugins/maps/public/classes/layers/layer.tsx

```diff
@@ -166,6 +170,14 @@ export class AbstractLayer implements ILayer {
     return this._descriptor;
   }
 
+  queryForTileMeta(mbMap: MbMap): Feature[] | null {
+    return null;
+  }
+
+  getMetaFromTiles(): Feature[] {
```

Comment:
Can "getMetaFromTiles" be removed from ILayer? Its only used privately by TiledVectorLayer.